### DISCUSSION TITLE
Do you think the llama model is stupid?

### DISCUSSION THREAD

**Discussion by makisekurisu-jp** [oobabooga/text-generation-webui#3318] [General]
![企业微信截图_20230726171551](https://github.com/oobabooga/text-generation-webui/assets/72290820/7d447190-4d65-441a-9d41-e13d0357dd32)
![企业微信截图_20230726171526](https://github.com/oobabooga/text-generation-webui/assets/72290820/ef04643b-a55d-47de-8ccc-1696f3bacaad)
![企业微信截图_20230726172432](https://github.com/oobabooga/text-generation-webui/assets/72290820/36cbc292-d91e-4827-9ae3-6f13f54ecade)

**Comment by BetaDoggo** [comment]
The issue is that you're running the 13B model with the lowest possible quant and prompting it like an instruct model. I'd recommend either changing your questions so that they will be answered through completion or switching to an instruct model like [Wizard](https://huggingface.co/TheBloke/WizardLM-13B-V1.2-GGML). I'd also recommend using the q4_K_M version if possible as it performs much better than q2_k.

**Reply by thesimpleone** [reply]
I would also recommend Llama 2.  They have both an instruct and a chat version.

**Comment by filmo** [comment]
I've run some tests with the Falcon 7b vs Llama 2 7b and it appears that Falcon 7b is better in my use case. 

Falcon 7b almost always creates coherent sentences with appropriate punctuation and structure. Llama 2 was failing to capitalize start words in sentences and also created some word salad.
